test/helpers/policygen/models.go

```diff
@@ -477,9 +480,9 @@ func (t *TestSpec) CreateCiliumNetworkPolicy() (string, error) {
 	}
 
 	type Spec struct {
-		EndpointSelector endpointSelector
```

Comment:
Why not use `pkg/policy/api/rule.go:Rule` directly?

Reply:
done.